techniques/system/distributePolicy/1.0/propagatePromises.cf

```diff
@@ -117,14 +117,21 @@ bundle agent propagatePromises
         classes => if_else("nodeslist_copied", "could_not_copy_nodeslist");
 
   methods:
+
+    policy_server.!root_server.nodeslist_copied::
+      "any" usebundle => service_reload("rudder-relayd");
```

Comment:
this will be done at each run: the if_else defines nodeslist_copied even if it was ok